modules/deepspace/assets/configs/playerSpawnConfig.json

```diff
@@ -1,23 +1,23 @@
 {
     "Alien Capital": {
-	    "hull": "deepspace:aliencapital",
+        "hull": "deepspace:aliencapital",
 	    "items": "core:advancedArmor core:advancedShield core:cannon-1 core:cannon-2 ",
 	    "money": 2000000
-    },
-	"Little Knight":{
-		"hull" : "deepspace:littleKnight",
-		"items": "99*rep core:advancedArmor core:advancedShield core:fixedGun 99*core:bulletClip",
-		"money": 10000
 	},
-	"Alien Attacker":{
-		"hull" : "deepspace:alienattacker",
-		"items": "99*rep core:advancedArmor-1 core:advancedShield-1 core:fixedGun-1 core:fixedGun-2 99*core:bulletClip",
-		"money": 10000
+    "Little Knight":{
```

Comment:
After I opened this thing locally, I noticed something - the whitespace is still off. Like, way off. Tabs everywhere.

Which editor do you use? If you use sublime/Atom, there should be an option to convert all tabs to spaces.

Reply:
um do spaces and tabs behave differently to the code ? I'm using intellij idea...

Reply:
They don't matter *functionally* (unless maybe in YAML or something...) but they matter a lot *visually* - tabs don't always display as the same width on different devices, and a mix of spaces and tabs in particular will tend to get messy fast. Then mix in different people having different settings and you quickly get to the point where it becomes a best practice to pick one very rigidly defined standard. Doesn't really matter if it is tabs *or* spaces but it has to be *one* and only one of them :-)